cli/src/opts/cast.rs

```diff
@@ -141,6 +141,24 @@ The input can be:
         #[clap(value_name = "VALUE")]
         value: Option<String>,
     },
+    #[clap(name = "<<")]
+    #[clap(visible_aliases = &["left-shift", "ls"])]
```

Comment:
As discussed in the issue the UX should be `cast shl <shift> <value>` (similar for right shift using `shr`). Would also discourage using `ls` as an alias - and generally I don't think we need any aliases for these when the names are `shl` and `shr`

Reply:
Wait, would that be `cast shl <shift> <value>` or `cast shl <value> <shift>` as my example ?